tensorboard/backend/application.py

```diff
@@ -467,7 +499,18 @@ def _serve_plugins_listing(self, request):
             if self._data_provider is not None
             else frozenset()
         )
-        for plugin in self._plugins:
+        plugins_to_consider = filter(
+            # Filter out experimental plugins that were not activated using the query param.
+            lambda plugin: (
+                plugin.plugin_name not in self._experimental_plugins
+            )
+            or (
+                plugin.plugin_name
+                in request.args.getlist(EXPERIMENTAL_PLUGINS_QUERY_PARAM)
+            ),
```

Comment:
Instead of `filter(lambda: ..., ...)`, use a list comprehension, per the
style guide:
<https://google.github.io/styleguide/pyguide.html#215-deprecated-language-features>

But probably simpler here to just conditionally `continue` at the top of
the `for`-loop? Certainly shorter:

```python
        plugins_to_skip = self._experimental_plugins - frozenset(
            request.args.getlist(EXPERIMENTAL_PLUGINS_QUERY_PARAM)
        )
        for plugin in self._plugins:
            if p.plugin_name in plugins_to_skip:
                continue
            ...
```